src/designer/elsa-workflows-studio/src/components/screens/workflow-blueprint-viewer/elsa-workflow-blueprint-viewer-screen/elsa-workflow-blueprint-viewer-screen.tsx

```diff
@@ -150,11 +151,17 @@ export class ElsaWorkflowBlueprintViewerScreen {
   renderCanvas() {
     return (
       <div class="elsa-flex-1 elsa-flex">
-        <elsa-designer-tree
+        {/* <elsa-designer-tree
           model={this.workflowModel}
           class="elsa-flex-1"
           ref={el => this.designer = el}
           mode={WorkflowDesignerMode.Blueprint}
+        /> */}
+        <elsa-designer
+          model={this.workflowModel}
+          class="canvas-container"
+          ref={el => this.designer = el}
+          mode={WorkflowDesignerMode.Blueprint}
```

Comment:
This needs to be somehow made configurable, otherwise everyone will suddenly use the new designer, whether they want to or not.